### ISSUE TITLE
Test Google AI

### ISSUE THREAD

**Issue by MenlyCSE** [STEM-E-Youth-Career-Development-Program/app-7#14]
Since Jiaming already has a way to use GPT's AI, with code examples, let's experiment with Google's AI... 

More info here: https://ai.google.dev/pricing
It's essentially the google version of ChatGPT. Test to see if you can use it with a webpage^^
If you can't use it for free, comment to inform us

[How it works (optional)](https://ai.google.dev/gemini-api/docs/api-overview#web_1)
[Embed into web app example (optional)](https://ai.google.dev/gemini-api/docs/get-started/tutorial?lang=web)

**Comment by R1tMah** [comment]
I tried using it to summarize this story below, but when i asked it with the prompt "summarize the story" it would respond with a few sentences, which I feel oversimplified the story. I then changed the prompt to "summarize the story to around 50 percent of its length" and it wouldn't rly listen until I clarified that it was still too short with a follow up prompt like "too short i said 50 percent"

First Response 
The passage explores the complex and contradictory nature of the color white. While often associated with positive qualities like purity, nobility, and joy, it also holds a hidden, unsettling power that evokes fear.
The author points to numerous cultural examples where white symbolizes power, goodness, and sacredness: from the white elephant in Asian royalty to the white robes of religious figures.
However, despite these positive associations, white ultimately holds a deeper, unsettling quality that strikes "panic to the soul," perhaps because of its association with blankness, emptiness, and the unknown.

Second response 
The passage explores the complex and often contradictory symbolism of whiteness. While it is associated with beauty, purity, royalty, and even divinity, it also harbors an unsettling, almost panicky undercurrent.
The author highlights numerous cultures and traditions that hold whiteness in high regard: the white elephant in Pegu and Siam, the white charger in Hanover, the white robe of priests, and the white garments of the redeemed in the Bible.
However, despite all these positive associations, a primal fear lingers within the concept of whiteness, something more unsettling than the fear of blood. This fear, the author suggests, is intrinsic to the hue itself, despite its widespread symbolism of goodness and power.

After follow up for second response 
The passage explores the complex and often contradictory symbolism of whiteness, highlighting how it is both revered and unsettling. The author points out numerous instances where whiteness holds a position of power and purity: the "Lord of the White Elephants" in Pegu, the white charger of the Hanoverian flag, and the white ermine robes of judges. Additionally, whiteness is often associated with the divine, exemplified by the Persian fire worshippers' reverence for the white forked flame, the Greek myth of Jove's white bull form, and the Iroquois's sacred White Dog sacrifice. Even in Christianity, the white alb worn by priests and the white robes of the redeemed in the Book of Revelation signify purity and holiness.
Despite this abundance of positive connotations, the passage argues that a deeper anxiety lingers within the color white. While blood evokes a primal fear, whiteness possesses an unsettling quality that strikes at the soul, a quality that is "elusive" and difficult to define. The author suggests that this underlying unease may be intrinsic to the color itself, existing alongside its numerous symbolic associations with beauty, power, and divinity.


"Though in many natural objects, whiteness refiningly enhances beauty, as if imparting some special virtue of its own, as in marbles, japonicas, and pearls; and though various nations have in some way recognised a certain royal preeminence in this hue; even the barbaric, grand old kings of Pegu placing the title ‘Lord of the White Elephants’ above all their other magniloquent ascriptions of dominion; and the modern kings of Siam unfurling the same snow-white quadruped in the royal standard; and the Hanoverian flag bearing the one figure of a snow-white charger; and the great Austrian Empire, Caesarian, heir to overlording Rome, having for the imperial colour the same imperial hue; and though this pre-eminence in it applies to the human race itself, giving the white man ideal mastership over every dusky tribe; and though, besides, all this, whiteness has been even made significant of gladness, for among the Romans a white stone marked a joyful day; and though in other mortal sympathies and symbolizings this same hue is made the emblem of many touching, noble things—the innocence of brides, the benignity of age; though among the Red Men of America the giving of the white belt of wampum was the deepest pledge of honour; though in many climes, whiteness typifies the majesty of Justice in the ermine of the Judge, and contributes to the daily state of kings and queens drawn by milk-white steeds; though even in the higher mysteries of the most august religions it has been made the symbol of the divine spotlessness and power; by the Persian fire worshippers, the white forked flame being held the holiest on the altar; and in the Greek mythologies, Great Jove himself being made incarnate in a snow-white bull; and though to the noble Iroquois, the midwinter sacrifice of the sacred White Dog was by far the holiest festival of their theology, that spotless, faithful creature being held the purest envoy they could send to the Great Spirit with the annual tidings of their own fidelity; and though directly from the Latin word for white, all Christian priests derive the name of one part of their sacred vesture, the alb or tunic, worn beneath the cassock; and though among the holy pomps of the Romish faith, white is specially employed in the celebration of the Passion of our Lord; though in the Vision of St. John, white robes are given to the redeemed, and the four-and-twenty elders stand clothed in white before the great-white throne, and the Holy One that sitteth there white like wool; yet for all these accumulated associations, with whatever is sweet, and honourable, and sublime, there yet lurks an elusive something in the innermost idea of this hue, which strikes more of panic to the soul than that redness which affrights in blood."
